docs/tomee-embedded-maven-plugin.adoc

```diff
@@ -16,772 +16,665 @@ using "classpath as war" option.
 It also allows to use a flat classpath deployment which is often use
 with microservices.
 
-tomee-embedded:run
+== tomee-embedded:run
 
 Full name:
 
-org.apache.tomee.maven:tomee-embedded-maven-plugin:7.0.0-M1:run
+- `org.apache.tomee.maven:tomee-embedded-maven-plugin:8.0.0-SNAPSHOT:run`
```

Comment:
Should we reference 8.0.0-M3 as opposed to the SNAPSHOT?

Reply:
Thanks Jon.  I've updated to M3 rather than SNAPHSOT.